curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98df.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-23
 
 # --description--
 
-You can add a fallback value to a variable by putting it as the second value of where you use the variable like this: `var(--variable-name, fallback-value)`. The property will use the fallback value when there's a problem with the variable. Add a fallback value of `green` to the `background-color` of `.bb2`.
+That didn't work, which is intended. You should add a fallback value to a variable by putting it as the second value of where you use the variable like this: `var(--variable-name, fallback-value)`. The property will use the fallback value when there's a problem with the variable. Add a fallback value of `green` to the `background-color` of `.bb2`.
```

Comment:
It's not intended... we have not yet explained what's going on.

Reply:
I am worried, that if we didn't indicate that we are expecting this to happen. people will spend time trying to fix it, then come to the forum for an issue that isn't there

Reply:
In that case it's to write in the previous step that the color will not change, here it's already a new step, the user know that that was expected, _it passed the tests_